pages/docs/adapters/fenics/adapter-fenics.md

```diff
@@ -5,15 +5,26 @@ keywords: adapter, fenics
 summary: "A general adapter for the open source computing platform FEniCS"
 ---
 
-## What is FEniCS ?
+## What is FEniCS
 
 From the FEniCS website: _FEniCS is a popular open-source (LGPLv3) computing platform for solving partial differential equations (PDEs). FEniCS enables users to quickly translate scientific models into efficient finite element code. With the high-level Python and C++ interfaces to FEniCS, it is easy to get started, but FEniCS offers also powerful capabilities for more experienced programmers. FEniCS runs on a multitude of platforms ranging from laptops to high-performance clusters._ More details can be found at [fenicsproject.org](https://fenicsproject.org/).
 
+## How to get FEniCS
+
+You can install FEniCS on your system by several ways as mentioned on [fenicsproject.org](https://fenicsproject.org/download/). The simplest way to install FEniCS on Ubuntu is using `apt-get`:
+
+```bash
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:fenics-packages/fenics
+sudo apt-get update
+sudo apt-get install fenics
```

Comment:
In (most of) the rest of the documentation, we use `apt install`, `apt update`, and `apt install`. The `apt-get` is an older command and I think there are no actively supported distributions still not supporting `apt`.

Side note: we should still use `apt-get` in scripts, as `apt` itselfs says it is not meant to be used in scripts at the moment (reports so when trying to run in a script).

```suggestion
sudo apt install software-properties-common
sudo add-apt-repository ppa:fenics-packages/fenics
sudo apt update
sudo apt install fenics
```

https://askubuntu.com/a/446484/142834